data_steward/cdr_cleaner/cleaning_rules/deid/fitbit_dateshift.py

```diff
@@ -48,8 +48,8 @@ def __init__(self, project_id, dataset_id, sandbox_dataset_id,
                          dataset_id=dataset_id,
                          sandbox_dataset_id=sandbox_dataset_id,
                          affected_tables=self.tables,
-                         map_dataset=map_dataset_id,
-                         map_table=map_tablename,
+                         mapping_dataset_id=mapping_dataset_id,
+                         mapping_table_id=DEID_MAP,
```

Comment:
@lrwb-aou is it safe to assume [at least for Erwin release] that any instance of `FitbitDateShiftRule` should have `mapping_table_id = "_deid_map"` or do we need to parameterize `mapping_table_id`?